src/test/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/resource/IncentiveReviewsResourceTest.kt

```diff
@@ -370,17 +375,6 @@ class IncentiveReviewsResourceTest : SqsIntegrationTestBase() {
     prisonApiMockServer.stubIepLevels()
     prisonApiMockServer.stubAgenciesIepLevels("MDI")
 
-    // pre-cache different next review dates as `persistPrisonerIepLevel` defaults lead to all being today + 1 year
-    nextReviewDateRepository.saveAll(
-      listOf(
-        NextReviewDate(bookingId = 1234134, nextReviewDate = iepTime.toLocalDate().plusDays(1)),
-        NextReviewDate(bookingId = 1234135, nextReviewDate = iepTime.toLocalDate().plusDays(2)),
-        NextReviewDate(bookingId = 1234136, nextReviewDate = iepTime.toLocalDate().plusDays(3)),
-        NextReviewDate(bookingId = 1234137, nextReviewDate = iepTime.toLocalDate().plusDays(4)),
-        NextReviewDate(bookingId = 1234138, nextReviewDate = iepTime.toLocalDate().plusDays(5)),
-      )
-    ).collect()
```

Comment:
no longer needed because last review dates are now set to be different in the test setup